tests/client_test.py

```diff
@@ -497,61 +497,6 @@ def testDeleteMultipleDocs(self):
 
         del self.Server['couchdbkit_test']
 
-    def testMultipleDocCOnflict(self):
```

Comment:
what's wrong with this test? Seems like it's just the last bit that should be removed (with `all_or_nothing`)

Reply:
good point https://github.com/dimagi/couchdbkit/pull/26/commits/8544e945d145f5b6ac7b8cd3452be1982355e120